CHANGELOG.rst

```diff
@@ -3,6 +3,16 @@ Changelog
 =========
 
 
+2.0.3 (unreleased)
+==================
+* Prevent changes to ``DJANGOCMS_LINK_XXX`` settings from requiring new
+  migrations
+* Changed naming of ``Aldryn`` to ``Divio Cloud``
+* Adapted testing infrastructure (tox/travis) to incorporate
+  django CMS 3.4 and dropped 3.2
+* Removed NetBios Pattern from documentations as its faulty
```

Comment:
I would replace "faulty" for "inaccurate" or "incorrect".
faulty is too generic